ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -803,6 +803,7 @@ periodics:
   reporter_config:
     slack:
       channel: '#announce-logging'
+      job_states_to_report: null
```

Comment:
I do not see this is relevant.
However none of presubmit failed.
/lgtm

Reply:
Please remove this change.